### ISSUE TITLE
Tooltips, Bootstrap and Screen Readers

### ISSUE THREAD

**Issue by vanch3d** [SAFeSEA/openEssayist-slim#14]
Bootstrap's tooltip are a good improvement on the defaut browser-dependent approach but are not suitable for screen readers.
The framework removes the title attribute and replace it by its own (data-original-title), making screen readers unable to access the title attribute anymore.
- adding ARIA tags (aria-label) is not recommended by specs, and override all textual information.
- JQueryUI tooltips seem to be much better (and are handling ARIA properly)
